app/src/org/commcare/tasks/ZipTask.java

```diff
@@ -223,7 +98,7 @@ private boolean zipFolder(File[] files, String zipFile, ZipOutputStream zos) thr
                 String fileName = pathParts[pathPartsLength - 1];
                 String fileFolder = pathParts[pathPartsLength - 2];
 
-                Log.d(TAG, "827 zipping folder with path: " + fileFolder + "/" + fileName);
+                Log.d(TAG, "Zipping instance folder with path: " + fileFolder + "/" + fileName);
 
                 ZipEntry entry = new ZipEntry(fileFolder + "/" + fileName);
                 zos.putNextEntry(entry);

```

Comment:
The docs for `putNextEntry` say that `closeEntry` should be called after the entry data has been written using `write`. `putNextEntry` does this for you if you forgot, but it won't for the last call entry that is added.